local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -18,6 +19,10 @@ apply plugin: "com.android.application"
  *   // whether to bundle JS and assets in debug mode
  *   bundleInDebug: false,
  *
+ *   // whether to bundle JS and assets in another build variant
+ *   // bundleInStageAlpha: true,
+ *   // bundleInBeta: true,
```

Comment:
@mkonicek I assume you are referring to this? I could change it to something more generic like bundleIn{variantName}